force-app/main/default/classes/RetriveCompletion.cls

```diff
@@ -0,0 +1,16 @@
+/*
+ * This class is used by copado JobExecution and publish the PE copado__Event__e
+ * 
+ *  Platform Event PE copado__Event__e
+ */
+global with sharing class RetriveCompletion implements copado.JobExecutionCallback {
```

Comment:
hard no.
please fix the class name's spelling and while you are at it, add the MCDO_ prefix to it:
MCDO_RetrieveCompletion


did we forget this component in the previous PR?

Reply:
Yes , part of previous one

Reply:
Also, did you Update the callback to this in the salesforce interface? I believe this was done on the function "Marketing Cloud Retrieve"